DependencyInjection/Compiler/ThemeCompilerPass.php

```diff
@@ -28,8 +28,12 @@ public function process(ContainerBuilder $container)
                 ->replaceArgument(2, null);
         }
 
-        $twigFilesystemLoaderDefinition = $container->getDefinition('twig.loader.filesystem');
+        $twigFilesystemLoaderDefinition = $container->findDefinition('twig.loader.filesystem');
         $twigFilesystemLoaderDefinition->setClass($container->getParameter('liip_theme.filesystem_loader.class'));
+        $twigFilesystemLoaderDefinition->setArguments(array(
```

Comment:
I am not totally sure, but is it possible that this call makes the bundle fail on Symfony 2.8.46? I receive an error like this:
```
Unable to find template "SuluWebsiteBundle:Analytics:website.html.twig|default".
```
But when i comment this call out, everything works again... Can somebody confirm? Should there be a check for the symfony version? Or does LiipThemeBundle starting from version 1.6 not support symfony 2 anymore?

Reply:
well we still support Symfony 2 if it doesn't cause major problems ..
I was expecting the test suite to catch any such issues, but maybe it does not ..

So if you can come up with a simple enough fix to make it work again I am happy to merge it.